airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/BasicAcceptanceTests.java

```diff
@@ -394,32 +401,65 @@ void testScheduledSync() throws Exception {
     final UUID operationId = testHarness.createOperation().getOperationId();
     final AirbyteCatalog catalog = testHarness.discoverSourceSchema(sourceId);
 
-    final ConnectionSchedule connectionSchedule = new ConnectionSchedule().units(1L).timeUnit(MINUTES);
     final SyncMode syncMode = SyncMode.FULL_REFRESH;
     final DestinationSyncMode destinationSyncMode = DestinationSyncMode.OVERWRITE;
     catalog.getStreams().forEach(s -> s.getConfig().syncMode(syncMode).destinationSyncMode(destinationSyncMode));
 
-    final var connection =
-        testHarness.createConnection(TEST_CONNECTION, sourceId, destinationId, List.of(operationId), catalog, connectionSchedule);
-
-    // When a new connection is created, Airbyte might sync it immediately (before the sync interval).
-    // Then it will wait the sync interval.
-    // if the wait isn't long enough, failures say "Connection refused" because the assert kills the
-    // syncs in progress
-    List<io.airbyte.api.client.model.generated.JobWithAttemptsRead> jobs = new ArrayList<>();
-    while (jobs.size() < 2) {
-      final var listSyncJobsRequest = new io.airbyte.api.client.model.generated.JobListRequestBody().configTypes(List.of(JobConfigType.SYNC))
-          .configId(connection.getConnectionId().toString());
-      final var resp = apiClient.getJobsApi().listJobsFor(listSyncJobsRequest);
-      jobs = resp.getJobs();
+    final UUID connectionId =
+        testHarness.createConnection(TEST_CONNECTION, sourceId, destinationId, List.of(operationId), catalog, ConnectionScheduleType.BASIC,
+            BASIC_SCHEDULE_DATA).getConnectionId();
+
+    for (int i = 0; i < 10; i++) {
       sleep(Duration.ofSeconds(30).toMillis());
+      try {
+        final JobRead jobInfo = testHarness.getMostRecentSyncJobId(connectionId);
+        waitForSuccessfulJob(apiClient.getJobsApi(), jobInfo);
+        break;
+      } catch (Exception e) {
+        // Retry.
+      }
     }
 
     testHarness.assertSourceAndDestinationDbInSync(WITHOUT_SCD_TABLE);
   }
 
   @Test
   @Order(9)
+  void testCronSync() throws Exception {
+    final UUID sourceId = testHarness.createPostgresSource().getSourceId();
+    final UUID destinationId = testHarness.createPostgresDestination().getDestinationId();
+    final UUID operationId = testHarness.createOperation().getOperationId();
+    final AirbyteCatalog catalog = testHarness.discoverSourceSchema(sourceId);
+
+    final ConnectionScheduleData connectionScheduleData = new ConnectionScheduleData().cron(
+        new ConnectionScheduleDataCron().cronExpression("* */2 * * * ?").cronTimeZone("UTC"));
```

Comment:
I'd add a comment explaining in human language what this cron expression means. I think this one is 'every 2 minutes', right?

This also makes me think it might be worth adding another test case, for a cron expression that we know *shouldn't* run any time soon, particularly since before we introduced cron schedules, a new scheduled connection always ran immediately